docs/pages/access-controls/compliance-frameworks/soc2.mdx

```diff
@@ -4,15 +4,10 @@ description: How to configure SOC 2-compliant access to SSH, Kubernetes, databas
 h1: SOC 2 Compliance for SSH, Kubernetes, Databases, Desktops, and Web Apps
 ---
 
-Teleport is designed to meet SOC 2 requirements for the purposes of accessing infrastructure, change management, and system operations. This document outlines a high
-level overview of how Teleport can be used to help your company to become SOC 2 compliant.
+Teleport is designed and independently audited to meet SOC 2 requirements for the purposes of accessing infrastructure, change management, and system operations. This document outlines a high level overview of how Teleport can be used to help your company to become SOC 2 compliant.
```

Comment:
This page is talking about how Teleport can help an organization become SOC 2 compliant, not how Teleport itself is SOC 2 audited.

Reply:
Yes -- however the page is the first result if you google "Teleport SOC 2" so we proably want to have some sort of pointer from it to the SOC 2 report.

I can make it a small snippet of custom text if you'd prefer instead of the template.